doc/tutorial/introduction.rst

```diff
@@ -0,0 +1,55 @@
+..
+    :copyright: Copyright (c) 2022 ftrack
+
+.. _tutorial/introduction:
+
+************
+Introduction
+************
+
+In this section, we are getting our hands dirty and showing an example on how the
+new framework can be customised to a studio specific needs.
+
+The aim with this exercise is to inspire the reader on what can be achieved with
+Connect - build a VFX studio pipeline that aids the artists to working the right
+way, with correct file naming convention, minimising tedious and error prone tasks.
+
+
+.. important::
+
+    There is currently no GUI approach to configuring the framework, and no way of
+    provide separate builds and configurations per project or context in general.
+    This is subject to be improved with the next major release of the framework.
+
+
+Checkout documentation code
+---------------------------
+
+The source code from this documentation, as fully working examples, can be found
+in the *resource* directory.
+
```

Comment:
@henriknorin-ftrack add how to build the docs while here ? :)

Reply:
Have added this to the developer section.